src/pages/documents/index.astro

```diff
@@ -0,0 +1,49 @@
+---
+import { getCollection } from "astro:content";
+
+import Layout from "@/layouts/base.astro";
+
+// Fetch all documents from the 'tags' collection
+let documents = await getCollection("documents");
+---
+
+<Layout class="p-6">
+    <div class="elements">
+        {
+            documents.map(doc => (
+                <a
+                    href={`/RbxDocs/documents/${doc.slug}`}
+                    class="inline-block rounded bg-blue-500 text-white hover:bg-blue-600"
+                    style={{
```

Comment:
200px ([12.5rem](https://nekocalc.com/px-to-rem-converter)) can be added to the Tailwind configuration if this measurement is absolutely needed. Will preview the branch shortly.

Other style properties like [`textAlign`](https://tailwindcss.com/docs/text-align), [`alignItems`](https://tailwindcss.com/docs/align-items) and [`justifyContent`](https://tailwindcss.com/docs/justify-content) can all be achieved with Tailwind.

Reply:
I'm not quiet sure how to use tailwind for the second part (can't seem to achieve the same result).

Reply:
This was meant to be highlighting the entire `style` property if this wasn't already obvious (I don't think it was, using a mouse is hard).

If by second part you're referring to align-items, `"items-center"` should be enough. Could you clarify on what exactly you're referring to?

Reply:
the second part I ment `align` and `justifyContent` styles

Reply:
`"items-center justify-center"` respectively as additional classes.